manageability-docker/TBH/setup

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
This file shouldn't be needed if using jrei/systemd-ubuntu:20.04  -- see https://hub.docker.com/r/jrei/systemd-ubuntu